docs/src/index.md

```diff
@@ -1,3 +1,35 @@
 # CLIMA
 
-Climate Machine
\ No newline at end of file
+```@meta
+CurrentModule = CLIMA
+```
+
+`CLIMA` is a software package that models the evolution of the Earth system over weeks to centuries.
+`CLIMA` solves partial differential equations that conserve the three-dimensional distributions of water, momentum, energy, 
+and tracers such as carbon between the Earth's atmosphere, ocean, and land surface.
+
+`CLIMA`'s purpose is to harness as much data as possible to predict droughts, heat waves, 
+rainfall extremes, and the evolution of Earth's climate.
+
+## Subcomponents
+
+`CLIMA` couples three models for the subcomponents of the Earth system:
+ 
+* `CLIMA.Atmos`: A model for the motion, transfer of solar radiation, and distribution of water, heat, and other tracers in the Earth's atmosphere.
+* `CLIMA.Ocean`: A model for the motion and distribution of temperature, salinity, and other tracers in the Earth's ocean.
```

Comment:
```suggestion
* `CLIMA.Ocean`: A model for the fluid mechanics and thermodynamics of ocean waters motion and the distribution of tracers  in it.
```

Reply:
I suppose if we say that `Clima.Ocean` is a model for "fluid mechanics", then we are referring to the fact that it is a discrete model for the continuous system? The way I think about it, one could think as the equations we solve as a model for motion. However, it is true that we solve discrete equations, rather than the continuous ones that are more properly the "model of fluid motion".

Reply:
I guess maybe I think of the equations themselves as the "mechanics". But divorcing myself from this view, perhaps the equations are a model for mechanics? This is the idea?

Reply:
Ok, I decided to make this change to refer to the code as a model for "mechanics", rather than "motion".